common/reqrepeater/standard.go

```diff
@@ -0,0 +1,50 @@
+package reqrepeater
+
+import (
+	"errors"
+	"time"
+)
+
+var ErrRetry = errors.New("try again later")
+
+// Strategy is configuration of how a connector handles faulty requests
+// connector will create Retry based on this config every time it attempts to make API call.
+type Strategy interface {
+	Start() Retry
```

Comment:
Connector can be in future configured on how request retries are performed.
For now there is only one `UniformRetryStrategy` and is hard coded for MS Sales with 3 retries every 1 sec.

Reply:
https://www.loom.com/share/39f1d4ea8dc446148eeff83a3f1c8681